tools/DfmHttpService/DfmHttpService.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmHttpService
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Build.Engine;
+    using Microsoft.DocAsCode.Plugins;
+
+    using Newtonsoft.Json;
+
+    public class DfmHttpService
+    {
+        private bool KeepGoing { get; set; } = true;
```

Comment:
good, I'll fix it

---
In reply to: [94039251](https://github.com/dotnet/docfx/pull/1075#discussion_r94039251) [](ancestors = 94039251,94037206)